core/psh/Makefile

```diff
@@ -4,11 +4,15 @@
 # Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-OBJS = $(filter-out psh psh-empties help, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh -name '*.c'))))
-PSH_COMMANDS ?= $(OBJS)
-OBJ_FEATURED = $(addsuffix .o,$(filter $(PSH_COMMANDS), $(OBJS)))
+ifdef PSH_DEFUSRPASSWD
+CFLAGS += -DPSH_DEFUSRPASSWD=\"$(PSH_DEFUSRPASSWD)\"
```

Comment:
I assume the intent is to compute the password hashed value in the `PSH_DEFUSRPASSWD` env variable in shell script somewhere? (or store it as a hash?)

Reply:
Yes. The intention was to be able to pass a hashed password to architectures with no filesystem thus no /etc/passwd file. The `PSH_DEFUSRPASSWD` should, at compile time, just store the password hash. I may change the name to `PSH_DEFUSRHASH` to represent that intent.

There was no intention of computing hash on compile time.

Reply:
Ack.